release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -845,8 +845,8 @@ You can now use the Special Resource Operator (SRO) to help manage the deploymen
 For more information, see xref:../hardware_enablement/psap-special-resource-operator.adoc#about-special-resource-operator_special-resource-operator[About the Special Resource Operator].
 
 [id="ocp-4-9-scale-memory-manager-in-beta-version"]
-==== Memory Manager feature (Technology Preview)
-The Memory Manager feature is now enabled by default for all pods running on the node that is configured with one of the following Topology Manager policies:
+==== Memory Manager is generally available
+The Memory Manager, a subcomponent of the Performance Addon Operator, is now enabled by default for all pods running on the node that is configured with one of the following Topology Manager policies:
```

Comment:
I don't think it is appropriate to say that Memory Manager is a subcomponent of the Performance Addon Operator. Memory manager is a Kubelet component and Performance Addon Operator is a way of configuring `kubeletconfig` with certain memory manager settings.

In addition to that, [Performance Addon Operator](https://github.com/openshift-kni/performance-addon-operators) has been deprecated and is moved to [Node Tuning Operator](https://github.com/openshift/cluster-node-tuning-operator) under that name of  referred to as [Performance Profile Controller](https://github.com/openshift/cluster-node-tuning-operator#perfomance-profile-controller).

Reply:
(just as a side note: NTO is what should be in the docs, performance profile controller or operator terminology is now internal only)

Reply:
Only PTO is mentioned in the 4.9, and that should be okay. The 4.10 version of this bug only affects the tech preview table that doesn't mention PTO or NTO or performance profile controller